src/main/java/org/icgc/argo/workflow/relay/service/IndexService.java

```diff
@@ -82,9 +82,9 @@ public IndexService(
     this.taskIndex = elasticsearchProperties.getTaskIndex();
   }
 
-  @SneakyThrows
+  //@SneakyThrows
```

Comment:
ugh I really dislike those, glad you've removed them. let's not leave cruft here, though
```suggestion
```